astropy/time/core.py

```diff
@@ -199,18 +196,20 @@ def __init__(self, val, val2=None, format=None, scale=None,
         else:
             self.location = None
 
-        if precision is not None:
-            self.precision = precision
-        if in_subfmt is not None:
-            self.in_subfmt = in_subfmt
-        if out_subfmt is not None:
-            self.out_subfmt = out_subfmt
-
         if isinstance(val, self.__class__):
```

Comment:
Fixed in 4eee963.  (Skipped CI tests because there are more changes in this branch coming).